relayer-cli/src/commands.rs

```diff
@@ -38,18 +38,17 @@ pub fn default_config_file() -> Option<PathBuf> {
     dirs_next::home_dir().map(|home| home.join(DEFAULT_CONFIG_PATH))
 }
 
-// TODO: Re-add the `config` subcommand
-// /// The `config` subcommand
-// #[options(help = "manipulate the relayer configuration")]
-// Config(ConfigCmd),
-
 /// Cli Subcommands
 #[derive(Command, Debug, Options, Runnable)]
 pub enum CliCmd {
     /// The `help` subcommand
     #[options(help = "Get usage information")]
     Help(Help<Self>),
 
+    /// The `config` subcommand
+    #[options(help = "Validate Hermes configuration file")]
+    Config(ConfigCmd),
```

Comment:
Can we make a `validate` subcommand for this? We may want to add other subcommands related to config, eg. `generate` so it might good to namespace it already.

Reply:
I think there is already a namespace for Config commands:


https://github.com/informalsystems/ibc-rs/blob/9c733d7af1ee7c2dee07086c24f2a94f4d169825/relayer-cli/src/commands/config.rs#L18

Is this what you meant?

Perhaps the command `help` is misleading because it only talks about validating configuration file; but I wrote it like that on purpose, since there is no other functionality yet. Once we add more subcommands (e.g., generate), we can rephrase the wording for the `help`. WDYT?

Reply:
Right, my bad! Yeah that's fine :)